src/ripple/app/tx/impl/SetSignerList.cpp

```diff
@@ -78,31 +88,77 @@ SetSignerList::determineOperation(
 NotTEC
 SetSignerList::preflight(PreflightContext const& ctx)
 {
-    auto const ret = preflight1(ctx);
-    if (!isTesSuccess(ret))
+    if (auto const ret = preflight1(ctx); !isTesSuccess(ret))
         return ret;
 
-    auto const result = determineOperation(ctx.tx, ctx.flags, ctx.j);
-    if (std::get<0>(result) != tesSUCCESS)
-        return std::get<0>(result);
-
-    if (std::get<3>(result) == unknown)
+    if (ctx.rules.enabled(featureSimplifiedSetSignerList) &&
```

Comment:
The "simplified" code path starting here largely duplicates most of the existing code path. Is it possible to de-duplicate that code to highlight the difference? What is the difference exactly?